sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -135,26 +135,34 @@ public UserDto selectActiveUserByLogin(DbSession session, String login) {
   }
 
   public List<UserDto> selectUsers(DbSession dbSession, UserQuery query) {
-    UserMapper mapper = dbSession.getMapper(UserMapper.class);
+    UserMapper mapper = getMapper(dbSession);
     return mapper.selectUsers(query);
   }
 
+  public long countRootUsersButLogin(DbSession dbSession, String login) {
+    return getMapper(dbSession).countRootUsersButLogin(login);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "l". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1488)